### PR TITLE
#26 - fix url validation

### PR DESCRIPTION
**Pull request by srinivasupadhya**

### CODE REVIEW

File: src/main/java/in/ashwanthkumar/gocd/github/util/URLUtils.java [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+import java.net.URL;
+import java.net.URLConnection;
+
+public class URLUtils {
+    public boolean isValidURL(String url) {
+        boolean isValid = true;
+        try {
+            URL urlObj = new URL(url);
+            URLConnection conn = urlObj.openConnection();
+            conn.connect();
```
Comment (ashwanthkumar) [inline src/main/java/in/ashwanthkumar/gocd/github/util/URLUtils.java]: @srinivasupadhya I haven't tried it but shouldn't we be closing the connection or does it happen automatically ?

Comment (srinivasupadhya) [inline src/main/java/in/ashwanthkumar/gocd/github/util/URLUtils.java]: isn't required i suppose - http://stackoverflow.com/questions/272910/in-java-when-does-a-url-connection-close

Comment (ashwanthkumar) [inline src/main/java/in/ashwanthkumar/gocd/github/util/URLUtils.java]: The second answer says if we cast it to `HttpURLConnection` then there's a `#disconnect()`. Let me do some benchmarking on number of CLOSE_WAIT connections when used on a large enough sample. That would answer if the sockets are getting closed or not.

Comment (srinivasupadhya) [inline src/main/java/in/ashwanthkumar/gocd/github/util/URLUtils.java]: sure. thanks :+1:

**Pull request merged**